macros/answerHints.pl

```diff
@@ -119,11 +136,13 @@ sub AnswerHints {
         checkCorrect => 0,
         replaceMessage => 0,
 	checkTypes => 1,
+        processPreview => $processPreview,
         score => undef,
         cmp_options => [],
         @options,
       );
-      next if $options{checkType} && $correct->type ne $student->type;
+      next if $options{checkTypes} && $correct->type ne $student->type;
```

Comment:
Note:  This fixes issue #379

Reply:
Yes. @dpvc fixed that bug along the way.

Reply:
Yeah,  I was just noting it so we remember to close that issue when this is merged.